src/platforms/android/configuration/gradle.mdx

```diff
@@ -260,13 +320,13 @@ sentry {
 
 ## ProGuard/R8 & DexGuard
 
-The plugin will automatically generate appropriate ProGuard mapping files and upload them when you run `gradle assemble{BuildVariant}`. For example, `assembleRelease` — Release is the default, but the plugin works for others if you have enabled ProGuard/R8. The credentials for the upload step are loaded through environment variables or from a `sentry.properties` file in your project root.
+The plugin will automatically generate appropriate ProGuard mapping files and upload them when you run `gradle assemble{BuildVariant}`. For example, `assembleRelease` — Release is the default, but the plugin works for others if you have enabled ProGuard/R8. The credentials for the upload step are defined in the Gradle extension above or loaded from a `sentry.properties` file in your project.
 
 If you are using the Guardsquare's ProGuard or DexGuard tooling, you have to enable the `experimentalGuardsquareSupport` flag.
 
 Learn more in the [full sentry-cli documentation](/product/cli/configuration/#configuration-values).
 
-At the very minimum, you will need something similar to:
+Alternatively to providing credentials via Gradle extension, you can create a `sentry.properties` file with the following contents under your `app` module:
```

Comment:
```suggestion
Instead of providing your credentials via the Gradle extension, you can create a `sentry.properties` file with the following content under your `app` module:
```